ch_util/rfi.py

```diff
@@ -39,23 +39,42 @@
 logger.addHandler(logging.NullHandler())
 
 
-# Ranges of bad frequencies given by their start and end frequencies (in MHz)
+# Ranges of bad frequencies given by their start time (in unix time) and corresponding start and end frequencies (in MHz)
+# If the start time is not specified, t = [], the flag is applied to all CSDs
 bad_frequencies = np.array(
     [
-        [449.41, 450.98],
-        [454.88, 456.05],
-        [457.62, 459.18],
-        [483.01, 485.35],
-        [487.70, 494.34],
-        [497.85, 506.05],
-        [529.10, 536.52],
-        [541.60, 554.49],
-        [564.65, 585.35],
-        [693.16, 693.55],
-        [694.34, 696.68],
-        [729.88, 745.12],
-        [746.29, 756.45],
-    ]
+        [[], [449.41, 450.98]],
+        [[], [454.88, 456.05]],
+        [[], [457.62, 459.18]],
+        [[], [483.01, 485.35]],
+        [[], [487.70, 494.34]],
+        [[], [497.85, 506.05]],
+        [[], [529.10, 536.52]],
+        [[], [541.60, 554.49]],
+        [[], [564.65, 585.35]],
+        [[], [693.16, 693.55]],
+        [[], [694.34, 696.68]],
+        [[], [729.88, 745.12]],
+        [[], [746.29, 756.45]],
+        [[], [505.85, 511.71]],  # 6 MHz band (reported by Simon)
+        [
+            [1633758888],
+            [584.00, 590.00],
+        ],  # from CSD 2893 (2021/10/09 - ) UHF TV Channel 33 (reported by Seth)
```

Comment:
Probably better for format like this:
```suggestion
        # from CSD 2893 (2021/10/09 - ) UHF TV Channel 33 (reported by Seth)
        [1633758888, [584.00, 590.00]],  
```